docker-build/docs/images.md

```diff
@@ -0,0 +1,76 @@
+
+# IBM Container Registry (ICR)
+WebSphere Application Server traditional [images](https://www.ibm.com/docs/en/was/9.0.5?topic=container-websphere-application-server-images) are available from the IBM Container Registry (ICR) at `icr.io/appcafe/websphere-traditional`. Our recommendation is to use ICR instead of Docker Hub since ICR doesn't impose rate limits on image pulls. Images can be pulled from ICR without authentication.
+
+The images for the latest 3 Websphere releases for 8.5.5.x and 9.0.5.x are available and are refreshed regularly (every 1-2 weeks) to include fixes for the operating system (OS). 8.5.5.x images are not available on the s390x platform.
+
+Available image tags are listed below. The tags use the following naming convention. 
```

Comment:
Users should pull images using the manifest only. So we should exclude tags with arch. We should also exclude all `ubi8` references (above and below) since we plan to provide images with only one OS (ubi8 or ubi9, but not both). 

So the `The tags use the following naming convention.` and the pattern below can be removed.